templates/base-supa-openapi/drizzle.config.ts

```diff
@@ -3,11 +3,17 @@ import { defineConfig } from "drizzle-kit";
 
 config({ path: "./.dev.vars" });
 
+const DATABASE_URL = process.env.DATABASE_URL;
+if (!DATABASE_URL) {
+  throw new Error("Missing Environment Variable: DATABASE_URL");
+}
+
 export default defineConfig({
   schema: "./src/db/schema.ts",
-  out: "./supabase/migrations",
+  out: "./drizzle/migrations",
```

Comment:
```suggestion
  out: "./supabase/migrations",
```

this path is specific to the supabase setup - a supabase app keeps its migrations in the supabase/migrations folder

Reply:
interesting. seemed to work fine with the adjustment, at least locally

Reply:
i guess we're not clear either considering this template was for a supabase hackathon, and it'd ultimately be dealer's choice 

so yes, handing over to the `drizzle` folder would make more sense if you're handing over full control to drizzle... if you're doing some sort of "dual control" between supabase and drizzle i'm not sure where that lands you. 

also not totally clear if keeping the migrations in the supabase folder affects anything in their studio... 

maybe indeed for the sake of aesthetic uniformity, you can survive and thrive on this hill. if we get any feedback on changing it, we can revisit